kbase-extension/static/kbase/js/widgets/appWidgets2/validators/resolver.js

```diff
@@ -44,9 +48,25 @@ define(['require', 'bluebird', 'widgets/appWidgets2/validation'], (require, Prom
         });
     }
 
+    function validateArray(values, parameterSpec, options) {
```

Comment:
If the validator has an array form, use it (only one for now), otherwise, just loop over them in the normal validate function.